setup.py

```diff
@@ -32,21 +32,19 @@
 # > requirements_list is a list of requirement; e.g. ['requests==2.6.0', 'Fabric==1.8.3']
 requirements_list = [str(ir.req) for ir in install_reqs]
 
+
 setup(
   name='fiware-glancesync',
-  packages=find_packages(exclude=['*tests*']),
   install_requires=requirements_list,
-  data_files=[('.', ['glancesync/settings/glancesync.conf'])],
-  package_data={
-    'glancesync': ['glancesync.conf']
-  },
-  include_package_data=True,
+  data_files=[('.', ['fiwareglancesync/settings/glancesync.conf']), ('.',
```

Comment:
It's not necessary because as you can see (https://github.com/telefonicaid/fiware-glancesync/pull/68/files#diff-ebacf6f6ae4ee68078bb16454b23247dL15) the files are copied to /etc